_config.yml

```diff
@@ -0,0 +1 @@
+theme: jekyll-theme-leap-day
```

Comment:
Since we're not using GitHub pages, I would prefer that we not add a theme setting for GitHub pages.  It may mislead authors to think that the theme will be honored.

Can you explain further what this file provides for the jenkins.io site?

Reply:
I was following the [Jenkins Contributing Guide](https://github.com/jenkins-infra/jenkins.io/blob/master/CONTRIBUTING.adoc#deploying-jenkinsio-on-github-pages).  BTW we may want to put a more explicit note, specifying that Deploying jenkins.io on GitHub pages is not currently supported.

In general "jekyll-theme-leap-day" is one the themes, provided by [jekyllrb](https://jekyllrb.com/docs/). Those themes allow to change the style of web documentation without changing it's content.

Reply:
I had just noticed a warning about "Rudimentary support" - so we do have such explicit note  on Jenkins Contributing Guide

Reply:
Yes, i agree, it looks that file is a GitHub Pages theme file